diff --git a/node_modules/agora-rn-uikit/src/RtcConfigure.tsx b/node_modules/agora-rn-uikit/src/RtcConfigure.tsx
index a6329ac..8a967e7 100644
--- a/node_modules/agora-rn-uikit/src/RtcConfigure.tsx
+++ b/node_modules/agora-rn-uikit/src/RtcConfigure.tsx
@@ -239,7 +239,7 @@ const RtcConfigure: React.FC<PropsWithChildren<Partial<RtcPropsInterface>>> = (
             }}>
             <MaxUidProvider value={uidState.max}>
               <MinUidProvider value={uidState.min}>
-                {rtcChannelJoined ? props.children : <React.Fragment />}
+                {props.children}
               </MinUidProvider>
             </MaxUidProvider>
           </RtcProvider>
